packages/documentation-site/patternfly-docs/pages/home.js

```diff
@@ -187,6 +187,7 @@ const HomePage = () => (
             <Content
               component={ContentVariants.p}
               className="pf-v6-u-font-size-lg"
+              isEditorial
```

Comment:
This element has the utility class `className="pf-v6-u-font-size-lg"`, which will override `isEditorial` - want to try removing the utility class font-size and see if the `isEditorial` styling gives either the same or an appropriate font-size?